server/server_codec_test.go

```diff
@@ -0,0 +1,142 @@
+package server
+
+import (
+	"reflect"
+	"testing"
+)
+
+func Test_RegisterDefaultMethodForNamespace(t *testing.T) {
+	tt := []struct {
+		name      string
+		namespace string
+		method    string
+		parser    Parser
+	}{
+		{name: "RegisterDefaultMethodForNamespace",
+			namespace: "multicast",
+			method:    "MulticastService.DefaultMethod",
+			parser:    new(MulticastArgsParser),
+		},
+		{name: "RegisterDefaultMethodForNamespace",
```

Comment:
You probably need a different name for the test here so if it fails we know which test fails.